hera_qm/tests/test_utils.py

```diff
@@ -404,3 +404,16 @@ def test_dynamic_slice():
     nt.assert_equal(b.shape, (1, 5))
 
     nt.assert_raises(ValueError, utils.dynamic_slice, 'foo', slice(0, None))
+
+
+def test_strip_extension():
+    path = 'goo/foo.boo/hoo/woo.two'
+    root = utils.strip_extension(path)
+    nt.assert_equal(root, path[:-4])
+
+
+def test_strip_extension_return_ext():
+    path = 'goo/foo.boo/hoo/woo.two'
+    root, ext = utils.strip_extension(path, return_ext=True)
+    nt.assert_equal(root, path[:-4])
```

Comment:
This assert is _technically_ redundant since it is executed in the previous test and all this test is supposed to do is test the extension returned by the helper function is correct. Or if you want to be really pedantic about if you could do this and rename the test to `test_strip_extension_return_ext_basename`
```suggestion
    nt.assert_equal(root, path[:-4])

def test_strip_extension_return_ext_extension():
    path = 'goo/foo.boo/hoo/woo.two'
    root, ext = utils.strip_extension(path, return_ext=True)
```